### ISSUE TITLE
UUID is not exported as default

### ISSUE THREAD

**Issue by impleri** [rse/pure-uuid#12]
In the TypeScript definition, UUID is marked as the default export. However, the UMD file does not provide it as the default, so the ES6+ import statement (`import UUID from 'pure-uuid') fails despite passing TS checking. Instead, the import statement has to aim a bit wider (`import * as UUID from 'pure-uuid'`). Can either the definition be corrected or the module export a default property aliasing the actual export so that both the TS definition and actual export mesh?
https://github.com/rse/pure-uuid/blob/d18a7578514caa5d0e8766a44ceb3bcefdad859d/uuid.d.ts#L61

**Comment by rse** [comment]
What makes you think that the UMD wrapper of Pure-UUID does NOT export the API as the default? It does export it as the default for both AMD and CommonJS and Browser environments according to the code. 

It works fine for both Babel...

```
$ cat test.js
import UUID from "pure-uuid"
console.log(new UUID(1).format())
$ npm install babel-cli babel-presets-env
$ npx babel-node --presets env test.js
94147450-ac02-11e7-bdde-3b17379e2e4f
```

...and plain Node.js environments:

```
$ cat test.js
const UUID = require("pure-uuid")
console.log(new UUID(1).format())
$ node test.js
```

What environment are you using for loading Pure-UUID?

**Comment by impleri** [comment]
I'm using TypeScript in Node.

For UUID to work, I have to use `import * as UUID from 'pure-uuid'`, but the d.ts indicates it should be `import UUID from 'pure-uuid'`. The UMD wrapper returns the module as an ES5 module (in which there is no actual _default_ export. It seems that the easiest thing is to alias `exports['default'] = UUID;` so that ES6 imports can see the root `module.exports` as _default_ as well.

**Comment by rse** [comment]
Now fixed in 1.4.9. Thanks.

**Comment by ArainUsman** [comment]
import { v4 as uuidv4 } from 'uuid';

solved my problem
Thanks
